ProcessLib/HT/CreateHTProcess.cpp

```diff
@@ -115,6 +115,25 @@ std::unique_ptr<Process> createHTProcess(
     DBUG("Use \'%s\' as specific_heat_capacity_fluid parameter.",
          specific_heat_capacity_fluid.name.c_str());
 
+
+    // Parameter for the thermal conductivity of the solid (only one scalar per
+    // element, i.e., the isotropic case is handled at the moment)
+    auto& thermal_dispersivity_longitudinal = findParameter<double>(
```

Comment:
I see that is a dispersion conductivity. Then the thermal conductivity is anisotropic. That means that  the conductivity parameter for the Laplacian has to be set as matrix.